server/templates/version.html

```diff
@@ -58,5 +70,4 @@
       </tbody>
     </table>
   </div>
-  <script src={{url_for('static', filename='translator.js', t=config['VERSION'])}}></script>
```

Comment:
is this relevant to this change?

Reply:
This was an old bug. The version page should not include translator js